homeassistant/components/seventeentrack/sensor.py

```diff
@@ -15,6 +15,8 @@
 REQUIREMENTS = ['py17track==2.2.2']
 _LOGGER = logging.getLogger(__name__)
 
+DOMAIN = 'seventeentrack'
```

Comment:
The domain should always be defined in the component or in a const module and imported into the component. It's confusing to define it in the platform. The entities generated by this platform will belong to the sensor domain.

Reply:
Does c8e6083 make more sense?